lib/rubygems/errors.rb

```diff
@@ -20,6 +20,41 @@ class LoadError < ::LoadError
     attr_accessor :requirement
   end
 
+  class MissingSpecError < Gem::LoadError
+    def initialize name, requirement
```

Comment:
Historically, RubyGems has followed Seattle Style and avoided parentheses wherever possible. At some point soon, we'll need to get consensus from the current maintainers and set up RuboCop so that we don't have to have discussions like this anymore. In the meantime, I'm okay merging either way, since we don't have a clear policy at the moment.